app/views/content/train-to-be-a-teacher/what-is-qts.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: "Qualified teacher status (QTS)"
 heading: "What is qualified teacher status (QTS)?"
-subcategory: Qualifications you can get
+subcategory: Qualifications
 description: |-
   Qualified teacher status (QTS) is what you need to teach in maintained primary, secondary and special schools in England. Find out how to get QTS.
 date: "2021-11-01"
 backlink: "../../"
 related_content:
-    What qualifications do you need to be a teacher? : "/is-teaching-right-for-me/qualifications-you-need-to-teach"
+    What qualifications do you need to be a teacher? : "/train-to-be-a-teacher/qualifications-you-need-to-teach"
```

Comment:
Does the link text need updating to not be a question now?

Reply:
leave as is